kuma/wiki/templates/wiki/document.html

```diff
@@ -73,6 +73,25 @@
   <meta property="og:description" content="{{ seo_summary }}">
   <meta name="description" content="{{ seo_summary }}">
   <meta name="twitter:description" content="{{ seo_summary }}">
+
+  <!-- Scrollback widget -->
+  <script>
+    var scrollbackAdded = false;
+  	function addScrollback(roomName) {
+      if (!roomName) return;
+      if(!scrollbackAdded) {
+    		window.scrollback = {"room": roomName,"form":"toast","minimize":false, titlebarColor: "#00539f"};
+        var el = document.createElement("script");
+        el.async = 1;
+        el.src = (location.protocol === "https:" ? "https:" : "http:") + "//scrollback.io/client.min.js";
+        document.getElementsByTagName("script")[0].parentNode.appendChild(el);
+        scrollbackAdded = true;
+      } else {
+        var iframe = document.getElementsByClassName("scrollback-stream")[0];
+        iframe.src = iframe.src.replace(/\/[^/]*\?/, '/' + encodeURIComponent(roomName) + '?');
+      }
+  	}
+  </script>
```

Comment:
@groovecoder Yes, that makes sense. Also, how do I get the username of the user who is logged into Kuma within the template? I need this to implement single sign on, where the user gets the same Scrollback username as his username in Kuma.